src/sentry/api/event_search.py

```diff
@@ -831,7 +832,7 @@ def convert_search_filter_to_snuba_query(search_filter, key=None):
             return condition
 
 
-def get_filter(query=None, params=None):
+def get_filter(query=None, params=None, adjust_interval=5):
```

Comment:
I really don't like having this new parameter this high up the call stack. Is there a reason we need to pass this in at all? What is stopping you from changing the hardcoded +5/-5 to a hardcoded +1/-1?

Reply:
Yeah, it's not very elegant, but hard coding this would change existing behavior outside of discover/performance and that seems undesirable.